File: 2833-furthest-point-from-origin/2833-furthest-point-from-origin.cpp
class Solution {
public:
    int furthestDistanceFromOrigin(string moves) {
        int right = 0;
        int countDash = 0;
        
        for(char c : moves) {
            if(c == '_')    countDash++;
            else if(c == 'R')   right++;
            else    right--;
        }
        
        return abs(right) + countDash;
    }
};

File: 2833-furthest-point-from-origin/2833-furthest-point-from-origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int right = 0;
        int dashed = 0;
        for(int i=0; i<moves.length(); i++) {
            if(moves.charAt(i) == '_') {
                dashed++;
            } else if(moves.charAt(i) == 'R') {
                right++;
            } else {
                right--;
            }
        }
        return Math.abs(right) + dashed;
    }
}